hal/src/main/native/athena/DigitalInternal.cpp

```diff
@@ -165,3 +191,8 @@ bool remapDigitalSource(HAL_Handle digitalSourceHandle,
 }
 
 }  // namespace hal
+
+// Unused function here to test template compile.
+__attribute__((unused)) static void CompileFunctorTest() {
+  hal::UnsafeManiputlateDIO(0, nullptr, [](hal::DIOSetProxy& proxy) {});
```

Comment:
Typo? `Maniputlate`.  Also, is this needed?

Reply:
This is needed as somewhere to instantiate the template and ensure it compiles. The unit test executable doesn't get compiled for athena.